Gemfile.lock

```diff
@@ -566,9 +570,9 @@ GEM
       method_source
       rake (>= 12.2)
       thor (~> 1.0)
-    rainbow (3.0.0)
+    rainbow (3.1.1)
     rake (13.1.0)
-    rbtree3 (0.6.0)
+    rbtree3 (0.7.1)
```

Comment:
### [rbtree3](https://github.com/kyrylo/rbtree3)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.6.0 → 0.7.1

[_[change-log](https://github.com/kyrylo/rbtree3/blob/master/ChangeLog), [source-code](https://github.com/kyrylo/rbtree3), [gem-diff](https://my.diffend.io/gems/rbtree3/0.6.0/0.7.1)_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/kyrylo/rbtree3/compare/v0.6.0...v0.7.1).

These are the first 10 commits:
- (69fc788) [Replace CircleCI with GitHub Actions](https://github.com/kyrylo/rbtree3/commit/69fc7881d85cdd6de2224932bcfdea6db0d2b41b)
- (1455a76) [.github/workflows: test against Ruby 3.0](https://github.com/kyrylo/rbtree3/commit/1455a769f805bd9663421d3c35429b32b18c0c02)
- (850e9d7) [Merge pull request #10 from kyrylo/gh-actions-instead-of-cir…](https://github.com/kyrylo/rbtree3/commit/850e9d76c22b0766bc2bc36987b11aed05eb2bb4)
- (0480b9e) [test.rb: rename duplicate `test_marshal` test names](https://github.com/kyrylo/rbtree3/commit/0480b9e197e47011702d18da5ad07628335e57f0)
- (bc55ca9) [Merge pull request #12 from kyrylo/test-renaming](https://github.com/kyrylo/rbtree3/commit/bc55ca97f2e939127bae3c5f237f9c2b68f62eea)
- (8f7feb8) [rbtree.c: delete rb_gc_force_recycle as it's deprecated in R…](https://github.com/kyrylo/rbtree3/commit/8f7feb8df8e65a3f7e8b5f9fb88ea0fa9204dee8)
- (e1f5e46) [Merge pull request #13 from kyrylo/gc-force-recycle-removal](https://github.com/kyrylo/rbtree3/commit/e1f5e46f14306b0cab9c0e9636fe7bb838e64bbe)
- (b867519) [rbtree.c: inherit from `rb_cObject` instead of `rb_cData`](https://github.com/kyrylo/rbtree3/commit/b8675193fed000409da7e939bf27aae03c4c6826)
- (d62618c) [Merge pull request #14 from kyrylo/rb-c-data](https://github.com/kyrylo/rbtree3/commit/d62618c5d6c7b00d0b2f7784dd73fe9ad46fab72)
- (a205a07) [.github/workflows/test: test against 3.1 & head](https://github.com/kyrylo/rbtree3/commit/a205a075943baaf82d95efd42ec5435f80db2bc0)


</details>